src/app/containers/ConsentBanner/Canonical/logic/index.js

```diff
@@ -0,0 +1,82 @@
+import Cookie from 'js-cookie';
+import setCookieOven from './setCookieOven';
+
+const PRIVACY_COOKIE = 'ckns_privacy';
+const EXPLICIT_COOKIE = 'ckns_explicit';
+const POLICY_COOKIE = 'ckns_policy';
+const COOKIE_EXPIRY = 365;
+const BANNER_APPROVED = '1';
+const POLICY_APPROVED = '111';
+const POLICY_DENIED = '000';
+
+const onClient = typeof window !== 'undefined';
+
+const setCookie = (name, value) =>
+  Cookie.set(name, value, { expires: COOKIE_EXPIRY });
+
+const setPolicyCookie = (value, logger) => {
+  setCookie(POLICY_COOKIE, value);
+  setCookieOven(POLICY_COOKIE, value, logger);
+};
+
+const seenPrivacyBanner = () => Cookie.get(PRIVACY_COOKIE) === BANNER_APPROVED;
+const seenCookieBanner = () => Cookie.get(EXPLICIT_COOKIE) === BANNER_APPROVED;
+const policyCookieSet = () => !!Cookie.get(POLICY_COOKIE);
+
+const setSeenPrivacyBanner = () => setCookie(PRIVACY_COOKIE, BANNER_APPROVED);
+const setDefaultPolicy = logger => setPolicyCookie(POLICY_DENIED, logger);
+const setAppovedPolicy = logger => setPolicyCookie(POLICY_APPROVED, logger);
+const setDismissedCookieBanner = () =>
+  setCookie(EXPLICIT_COOKIE, BANNER_APPROVED);
+
+const canonicalBannerLogic = ({
```

Comment:
Likewise as stated in the other file, making this function reflect what it is doing would be useful
```suggestion
const consentBannerUtilities = ({
```

Reply:
Done :)